### DISCUSSION TITLE
Consistent File Naming

### DISCUSSION THREAD

**Discussion by leumasme** [PrismarineJS/mineflayer#1728] [General]
As it currently is, filenames in the Project File Structure are a mess between lowerCamelCase.js and snake_case.js (and... this-weird-case.js ?)
See

examples/
armor_stand.js
auto-eat.js
chatAddPattern.js

lib/
promise_util.js
supportFeature.js

all tests/ files are also in lowerCamelCase, but this might just be the convention for tests?

You should see what i mean.
This is quite a mess, and i believe this should be cleaned up, but this would be quite annoying for anyone currently working on these files.
What are your thoughts on this?

**Comment by rom1504** [comment]
We can rename them.
Git handle file renames fine, there shouldn't be much conflicts.

If you're interested by this topic, can you check what's the recommended /
most common naming scheme in node ?
